src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -346,8 +346,18 @@ private void WriteHeader()
 
         private void WriteHelp()
         {
-            //TBD
-            Console.WriteLine("In Development");
```

Comment:
We should have name and version as part of output header. For example,

```
Microsoft .NET Core Xml Serialization Generation Utility
Version: 1.0.0-preview1-xxxx
```

I am wondering, what's the message if someone runs this tool while .NET Core 2.0 is not installed on the machine.